### ISSUE TITLE
About Val

### ISSUE THREAD

**Issue by celiazzx** [daooshee/HLA-Face-Code#11]
Hi, did you do the verification. If so how was the validation set organized

**Comment by daooshee** [comment]
Hi, you can refer to [this issue](https://github.com/daooshee/HLA-Face-Code/issues/5#issuecomment-885554738).

**Comment by celiazzx** [comment]
It seems that your training process does not do validation. When I run val_ss.py I get an error: 'Error in WIDERDetectionTransfer: XXX not found', I think I should also need a SRC_VAL_FILE, which corresponds to wider original and wider add noise respectively

**Comment by daooshee** [comment]
> It seems that your training process does not do validation.

Yes, I split training and validation for convenience. I first save many checkpoints (each [2500](https://github.com/daooshee/HLA-Face-Code/blob/2b5737e206741378dcc599ee3fd5b3c6198c8705/train_code/train.py#L527) iteration) and then do validation to find the best one.

> When I run val_ss.py I get an error: 'Error in WIDERDetectionTransfer: XXX not found', I think I should also need a SRC_VAL_FILE, which corresponds to wider original and wider add noise respectively

It indeed needs WIDER FACE validation set with noise. I will try to fix this problem and upload the WIDER FACE validation set as soon as possible. You can also create this set using released models following [./train_noise_synthesis](https://github.com/daooshee/HLA-Face-Code/blob/main/train_code/train_noise_synthesis/README.md) or use `SRC_TRAIN_FILE` to replace `SRC_VAL_FILE` in [val_loader_ss_consrc](https://github.com/daooshee/HLA-Face-Code/blob/2b5737e206741378dcc599ee3fd5b3c6198c8705/train_code/val_ss.py#L125).
